### PR TITLE
Update install for VS 15

### PR DESCRIPTION
**Pull request by zooba**
Changes from fopx installer to swix for use with VS "15".
This also requires updating project files to look for targets slightly differently, which is why there are changes to the core product.

### CODE REVIEW

File: Python/Tests/Core/ProjectUpgradeTests.cs [hunk 1]
```diff
@@ -291,79 +291,57 @@ out factoryGuid
...
             var factory = new PythonProjectFactory(null);
             var sp = new MockServiceProvider();
-            var shell = new MockVsShell();
-
             sp.Services[typeof(SVsQueryEditQuerySave).GUID] = null;
             sp.Services[typeof(SVsActivityLog).GUID] = new MockActivityLog();
-            sp.Services[typeof(SVsShell).GUID] = shell;
             factory.Site = sp;
 
-            var projectFile = TestData.GetPath(Path.Combine("TestData", "ProjectUpgrade", "WebProjectType.pyproj"));
```
Comment (huguesv) [inline Python/Tests/Core/ProjectUpgradeTests.cs]: WebProjectType.pyproj is no longer used?  If so you could delete the file.

File: Python/Setup/swix/file_associations.swr [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    
+    vs.fileAssociation
+        Extension=.pyw
+        ProgId=VisualStudio.py.[InstanceId]
+        DefaultProgramRegistrationPath=$(VSDefaultProgramPath)
+    
+    vs.fileAssociation
+        Extension=.pyproj
+        ProgId=VisualStudio.Launcher.pyproj.[InstanceId]
+        PerceivedType=text
```
Comment (huguesv) [inline Python/Setup/swix/file_associations.swr]: what does this mean?

File: Python/Setup/swix/product.json [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        },
+        "microsoft.PythonTools.IronPython": {
+          "version": "[3.0.0.0, 4.0)"
+        },
+        "microsoft.PythonTools.IronPython.Templates": {
+          "version": "[3.0.0.0, 4.0)"
+        },
+        "microsoft.PythonTools.ML": {
+          "version": "[3.0.0.0, 4.0)"
+        }
```
Comment (huguesv) [inline Python/Setup/swix/product.json]: Uwp and Uwp.Templates are missing, is it intentional?

**Comment by huguesv** [comment]
:shipit:

File: Python/Setup/swix/file_associations.swr (hunk 2 above)
Comment (zooba) [inline Python/Setup/swix/file_associations.swr]: It means that the items in `HKCR\SystemFileAssociations\text` also apply to this file type, which is wrong in this case. Updating this to be `ContentType=application/xml` as it should be (and moving `PercievedType=text` onto the source file types).

File: Python/Setup/swix/product.json (hunk 3 above)
Comment (zooba) [inline Python/Setup/swix/product.json]: No, fixed.

**Pull request merged**